kolena/_extras/metrics/sklearn.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021-2023 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+try:
+    import sklearn.metrics as sklearn_metrics
+
+    sklearn_metrics.__name__
```

Comment:
I would remove this in favor of suppressing the error that this exists to quiet — something like `# noqa: F401` on the import line